src/PerformanceTests/Transport.V7.RabbitMQ.V5/RabbitMQProfile.cs

```diff
@@ -10,14 +10,15 @@ class RabbitMQProfile : IProfile, INeedPermutation
 
     public void Configure(EndpointConfiguration endpointConfiguration)
     {
-        var cs = ConfigurationHelper.GetConnectionString("RabbitMQ");
+        var cs = ConfigurationHelper.GetConnectionString("RabbitMQ2");
         var builder = new DbConnectionStringBuilder { ConnectionString = cs };
         if (builder.Remove("prefetchcount")) NServiceBus.Logging.LogManager.GetLogger(nameof(RabbitMQProfile)).Warn("Removed 'prefetchcount' value from connection string.");
 
         var transport = endpointConfiguration
             .UseTransport<RabbitMQTransport>()
             .UseConventionalRoutingTopology()
-            .PrefetchMultiplier(Permutation.PrefetchMultiplier);
+            .PrefetchMultiplier(Permutation.PrefetchMultiplier)
```

Comment:
As you've hardcoded this option, might we want to backport this to the previous major versions?

Reply:
I don't mind. Feel free to "backport" this